pkg/ocm/ocm_config.go

```diff
@@ -0,0 +1,136 @@
+package ocm
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+)
+
+// Config is the type used to store the configuration of the client.
+// There's no way to line-split or predefine tags, so...
+//nolint:lll
+type Config struct {
```

Comment:
are you sure we need this?

Reply:
Unfortunately yes, if we want to pull in the ocm-cli implementation as static code in our source.